notebooks/psf_photometry/NIRCam/nircam_spacephot.ipynb

```diff
@@ -0,0 +1,1376 @@
+{
```

Comment:
<p>I have the exact same comments as for the MIRI notebook, so I will not add any below. I think that many parts are not clear, the usage of WebbPSF is uncertain and some of the results are puzzling.</p><p> </p><p>NIRCam images can be severely undersampled, so astrometry and photometry with them need a careful treatment. WebbPSF PSFs also do not include detector effects. Before making everything public, the undersampling systematics should be understood. The result of the PSF fit seems not that bad, but just looking at the subtracted image is not enough.</p><p> </p><p>I would choose a different set of images. The background in all images is significantly negative.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thanks! I'll return to these comments on NIRCam at a later date. If you have any good images to use, please let me know.</p>